components/dashboard/src/components/SelectableCardSolid.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+export interface SelectableCardSolidProps {
+    title: string;
+    selected: boolean;
+    className?: string;
+    onClick: () => void;
+    children?: React.ReactNode;
+}
+
+function SelectableCardSolid(props: SelectableCardSolidProps) {
+    return (
+        <div
+            className={`rounded-xl px-3 py-3 flex flex-col cursor-pointer group transition ease-in-out ${
+                props.selected ? "bg-gray-800" : "bg-gray-100 dark:bg-gray-900 hover:bg-gray-200 dark:hover:bg-gray-800"
```

Comment:
**suggestion**: Just like how we highlight the most elavated option with dark color in the light theme we need to also do the same for the dark theme, see relevant comment in https://github.com/gitpod-io/gitpod/issues/7932#issuecomment-1026269854. Feedback is welcome, @mustard-mh!

| Light | Dark |
|-|-|
| <img width="526" alt="Frame 369" src="https://user-images.githubusercontent.com/120486/158707467-43e0b78f-46c0-4b16-8d37-58bb63e32a2e.png"> | <img width="526" alt="Frame 36942" src="https://user-images.githubusercontent.com/120486/158707469-5461a67a-3bf0-4656-817b-1eb7a8dd0e80.png"> |

I've tried suggesting some code changes towards this design but I could be missing something:

```suggestion
                props.selected ? "bg-gray-800 dark:bg-gray-100" : "bg-gray-100 dark:bg-gray-800 hover:bg-gray-200 dark:hover:bg-gray-700"
```

Reply:
Yeah, I cannot find the design page in Figma, so I just set some temporary colors😅

Reply:
**fyi**: @mustard-mh most of the times, when helpful or needed, detailed design specs are linked below the exported images. For example, see the link in the issue description for #8028.

| Exported and link to design specs in issue description for #8028 |
|-|
| <img width="838" alt="Screenshot 2022-03-17 at 11 09 11 AM" src="https://user-images.githubusercontent.com/120486/158775766-05bddcfb-6a63-4af5-bd57-4e85ae3401c3.png"> |

Reply:
**issue**: Unresolving this as the suggested code changes are still pending. @mustard-mh let me know if you have any feedback on these changes. In the screenshots in https://github.com/gitpod-io/gitpod/pull/8783/files#r828527752 the last three tiles describe how default, active, and hover state could look like in light and dark theme.